lib/release/notes/link.rb

```diff
@@ -35,7 +35,7 @@ def link_lines(lines:)
         def replace(line, issue_number, label, index)
           identifier = "#{label.split(/\s/)[0]} #{issue_number}"
           humanized = "#{link_to_humanize[index]} #{issue_number}"
-          linked = "[#{humanized}](#{link_to_sites[index]})"
+          linked = "[#{humanized}](#{link_to_sites[index]}\/#{issue_number[1..-1].scan(/^[^\[\]]*/)[0]})"
```

Comment:
Try `String#tr` or `String#gsub` - scan TMK (w/o benchmarking at the moment) is going to take a lot longer as it's returning an array of occurrences which is why your having to reference the first index in the array.
```suggestion
          linked = "[#{humanized}](#{link_to_sites[index]}\/#{issue_number.tr('^0-9', '')})"
```
or
```suggestion
          linked = "[#{humanized}](#{link_to_sites[index]}\/#{issue_number.gsub(/[^\d]/, '')})"
```

From a quick google search i believe tr is the fastest.